test/Infrastructure/LeanCode.AuditLogs.Tests/AuditLogsIntegrationTests.cs

```diff
@@ -202,7 +204,12 @@ IRegistrationContext context
 
     internal sealed class StubAuditLogStorage : IAuditLogStorage
     {
-        private readonly ILogger logger = Log.ForContext<StubAuditLogStorage>();
+        private readonly Logging.ILogger<StubAuditLogStorage> logger;
+
+        public StubAuditLogStorage()
+        {
+            logger = Substitute.For<Logging.ILogger<StubAuditLogStorage>>();
```

Comment:
We probably should have a `NullLogger<T>` implementation. Using NSubstitute for this gives a wrong impression that it is used anyhow.